src/embit/bip47.py

```diff
@@ -0,0 +1,43 @@
+from . import base58, bip32, bip39
+from .bip32 import HDKey
+from .networks import NETWORKS
+from .wordlists.bip39 import WORDLIST
+
+from io import BytesIO
+
+"""
+    BIP-47: https://github.com/bitcoin/bips/blob/master/bip-0047.mediawiki
+"""
+
+def get_payment_code(root: HDKey):
+    """
+        Generates the recipient's BIP-47 shareable payment code (version 1)
+        for the input root private key.
+    """
+    bip47_child = root.derive("m/47'/0'/0'")
```

Comment:
Shouldn't this derivation be customizable?
Looks like it should be more like `m/47'/<coin>'/<account>'` - without customizing derivation path one can't make more than one payment code from the root key, and that sucks because payment code is like an identity.

```suggestion
def get_payment_code(root: HDKey, coin=0, account=0):
    """
        Generates the recipient's BIP-47 shareable payment code (version 1)
        for the input root private key.
    """
    bip47_child = root.derive("m/47'/%d'/%d'" % (coin, account))
```

Reply:
Yes, you're right. Update will be in my next commit!